### PR TITLE
"Fix" for boundingSphere Radius

### PR DESCRIPTION
**Pull request by Pryme8**
A better approximation for BS radius.

### CODE REVIEW

**Comment by Pryme8** [comment]
I think this should fix the problem people are having with getting a true BoundingSphere radius.

File: src/Culling/babylon.boundingSphere.ts [hunk 1]
```diff
@@ -26,6 +27,21 @@
         public reConstruct(min: Vector3, max: Vector3) {
             this.minimum = min.clone();
             this.maximum = max.clone()
+            
+            var _min = Number.POSITIVE_INFINITY;
+            var _max = Number.NEGATIVE_INFINITY;
+            var _va = ['x','y','z'];
+    
+            for(var i=0; i<3; i++){
+                if(min[_va[i]]<_min){
```
Comment (deltakosh) [inline src/Culling/babylon.boundingSphere.ts]: this seems wrong to me: the distance cannot be on different axis. How the distance between x and z could work for isntance? we have to evaluate the overall distance

**Comment by jbousquie** [comment]
Not sure this should be merged because of this explanation : http://www.html5gamedevs.com/topic/38508-bounding-sphere-radius-and-point-intersection/?do=findComment&comment=219771

I think the current bsphere radius is just correct.

**Comment by Pryme8** [comment]
Gotcha, yeah It was a misunderstanding in what the bounding radius was.

**Pull request closed without merging**